packages/server/src/models/hooks.js

```diff
@@ -0,0 +1,27 @@
+const events = require('../async/events');
+const emitter = require('../async/emitter');
+
+const { MEASURABLE_STATE } = require('./enums/measurable-state');
+
+function addHooks(db) {
+  db.sequelize.addHook('afterUpdate', (instance) => {
```

Comment:
why are these here instead of in files related to the model, like models/measurable?

Reply:
The purpose is to encupsulate this part of application. Here "Models" just drivers to DB nothing more.